contrib/confluence/src/python/pants/contrib/confluence/util/confluence_util.py

```diff
@@ -0,0 +1,172 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import getpass
+import mimetypes
+import urllib
+from os.path import basename
+
+from twitter.common import log
+
+
+# Copied from `twitter.common.confluence`
```

Comment:
Should probably preserve the copyright statement of copied code: ie:
```
# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
# Copyright 2011 Twitter, Inc.
# Licensed under the Apache License, Version 2.0 (see LICENSE).
```

Reply:
This fails the style checks. I added it below the comment stating I copied it from twitter.common.